tools/data/sthv1/preparing_sthv1.md

```diff
@@ -7,16 +7,32 @@ Before we start, please make sure that the directory is located at `$MMACTION2/t
 
 First of all, you have to sign in and download annotations to `$MMACTION2/data/sthv1/annotations` on the official [website](https://20bn.com/datasets/something-something/v1).
 
-## Step 2. Prepare Videos
+## Step 2. Prepare RGB Frames / Videos
 
-Then, you can download all data parts to `$MMACTION2/data/sthv1/` and use the following command to extract.
+In Step 2, you have two choices: directly downloading RGB frames on [sthv1 website](https://20bn.com/datasets/something-something/v1) or downloading videos on [sthv2 website](https://20bn.com/datasets/something-something/v2) and extracting frames.
+
+### Plan A Prepare RGB Frames
+
+You can download all RGB frame parts on [sthv1 website](https://20bn.com/datasets/something-something/v1) to `$MMACTION2/data/sthv1/` and use the following command to extract.
 
 ```shell
 cd $MMACTION2/data/sthv1/
 cat 20bn-something-something-v1-?? | tar zx
 cd $MMACTION2/tools/data/sthv1/
 ```
 
+Afterwards, you can skip to step 4 to generate file lists in the format of rawframes. Since the prefix of official JPGs is "%05d.jpg"(e.g: 00001.jpg), you have to add "filename_tmpl='{:05d}.jpg'" to dictionaries data.train, data.val and data.test of the sthv1 config.
```

Comment:
Since the prefix of official JPGs is "%05d.jpg" (e.g., "00001.jpg"), you have to add "filename_tmpl='{:05d}.jpg'" to the dict of data.train, data.val and data.test in the config files related with sthv1 like this:
```python
data = dict(
    videos_per_gpu=6,
    workers_per_gpu=4,
    train=dict(
        type=dataset_type,
        ann_file=ann_file_train,
        data_prefix=data_root,
        filename_tmpl='{:05}.jpg',
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        ann_file=ann_file_val,
        data_prefix=data_root_val,
        filename_tmpl='{:05}.jpg',
        pipeline=val_pipeline),
    test=dict(
        type=dataset_type,
        ann_file=ann_file_test,
        data_prefix=data_root_val,
        filename_tmpl='{:05}.jpg',
        pipeline=test_pipeline))
```